conjur/init_logic.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+
+"""
+InitLogic module
+
+This module is the business logic for writing configuration information
+to the user's machine as well as fetching certificates from Conjur
+"""
+
+# Builtins
+import logging
+import sys
+import os.path
+from pathlib import Path
+
+# Third party
+import socket
+import yaml
+from OpenSSL import SSL
+from OpenSSL.crypto import FILETYPE_PEM
+
+# pylint: disable=raise-missing-from
+class InitLogic:
+    """
+    InitLogic
+
+    This class holds the business logic for population the
+    conjurrc configuration details needed to connect with Conjur
+    """
+    DEFAULT_CONFIG_FILE = os.path.expanduser(os.path.join('~', '.conjurrc'))
+    DEFAULT_PROMPT_RESPONSE=True
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def connect_to_socket(hostname, port):
```

Comment:
`connect_to_socket` isn't giving extra info with the trailing words. `connect` should be good enough.